packages/braid-design-system/src/lib/components/Drawer/Drawer.docs.tsx

```diff
@@ -38,6 +39,15 @@ const drawerPreviewPropsFromSourceValue = (element: DrawerElement) => ({
   scrollLock: false,
 });
 
+export const DrawerPreview = ({ children }: { children: ReactNode }) => (
```

Comment:
No longer needs to be exported as its no longer shared.
```suggestion
const DrawerPreview = ({ children }: { children: ReactNode }) => (
```